src/JWT/Serializers/JsonSerializerFactory.cs

```diff
@@ -1,16 +1,28 @@
 ﻿namespace JWT.Serializers
 {
-    internal static class JsonSerializerFactory
+    public static class JsonSerializerFactory
     {
-        public static IJsonSerializer CreateSerializer()
+        private static IJsonSerializer _serializer;
+
+        public static IJsonSerializer Serializer

```

Comment:
Would something like this better?

```csharp
public static IJsonSerializer Serializer { get; set; } = new ...
```

or

```csharp
public static IJsonSerializer Serializer => new ...
```

(I'm always confused between these two syntax options)

Reply:
Or better let's make it a method, a factory with property looks more like a nonsense :)

Reply:
Yeah, a method looks better in a factory.